Diana.lua

```diff
@@ -33,14 +36,19 @@ do
   end
 end
 
+-- Set key of 'Diana' to $param -> 'key'
 Diana.setKey = function(key)
   apiKey = '?api_key=' .. key;
 end
 
+-- Set region of 'Diana' to $param -> region
 Diana.setDefaultRegion = function(region)
   defaultRegion = region;
 end
 
+[[-- Get champion data.
```

Comment:
Block comments in Lua are `--[[ ]]--` instead of `[[-- --]]`

Reply:
Hmmm okay, still colored the syntax as if it still was a comment lol